data/json/monsters/monsters.json

```diff
@@ -532,6 +533,43 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_fused",
+    "type": "MONSTER",
+    "name": "fused",
+    "description": "Some human bodies fused together into a colossus with heads and limbs sticking out of its bloated body. You may have trouble estimating its healthiness and its capabilities might change.",
```

Comment:
I would omit some from this line, just start with `"Human bodies fused together into..."`